queue/message-queue-event.js

```diff
@@ -119,7 +121,7 @@ module.exports = exports = class MessageQueueEvent {
 
 	publish() {
 		return publish(this)
-			.then(success => success);
+			.then(success => success).catch((e) => this.log.error('publish queue event', e))
```

Comment:
**Question**: what is publish queue event? perhaps it would be more useful if the message was more explicit about what's failing?